app/controllers/questions/demographic_english_conversation_controller.rb

```diff
@@ -2,6 +2,8 @@ module Questions
   class DemographicEnglishConversationController < QuestionsController
     include AuthenticatedClientConcern
 
+    skip_before_action :redirect_if_completed_intake_present
```

Comment:
Probably better to have a `BaseDemographicQuestionController` or `PostCompletionQuestionsController`